keras/wrappers/scikit_learn.py

```diff
@@ -6,6 +6,18 @@
 from ..utils.np_utils import to_categorical
 
 
+class Hideout(object):
```

Comment:
I'm not sure it is the best way to proceed. It is definitely the best one I have though about. The most straightforward solution, which I didn't want to pursue, would be to change the name of the get_params method all over keras core code. At least in the current solution all the changes are confined to the scikit_learn.py file.

I would be happy to hear different approaches, if there are any.